Colours.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+
+namespace game
+{
+    internal class Colours
+    {
+
+
+        static public ConsoleColor GoldReward {get { return ConsoleColor.Yellow;}}
+        static public ConsoleColor Gold {get { return ConsoleColor.DarkYellow;}}
+        static public ConsoleColor Damage {get { return ConsoleColor.DarkRed;}}
+        public static ConsoleColor Health { get { return ConsoleColor.Green;}}
```

Comment:
static public or public static ... makes no difference, but should be the same, pick one.